server.js

```diff
@@ -28,9 +28,11 @@ app.put('/api/v1/favorites/:id', async (request, response) => {
     const { id } = request.params;
     const product = await database('product').where('id', '=', id);
```

Comment:
This will obtain the product from our database whose id matches the id from our request.params (line 28).